services/storage/src/simcore_service_storage/simcore_s3_dsm.py

```diff
@@ -1236,6 +1239,61 @@ async def _create_fmd_for_upload(
             fmd=fmd
         )
 
+    async def create_s3_export(
+        self,
+        user_id: UserID,
+        object_keys: list[StorageFileID],
+        *,
+        progress_cb: AsyncReportCB | None,
+    ) -> StorageFileID:
+        source_object_keys: set[StorageFileID] = set()
+
+        for object_key in object_keys:
+            async for meta_data_files in get_s3_client(self.app).list_objects_paginated(
+                self.simcore_bucket_name, object_key
+            ):
+                for entry in meta_data_files:
+                    source_object_keys.add(entry.object_key)
+
+        _logger.debug(
+            "User selection '%s' includes '%s' files",
+            object_keys,
+            len(source_object_keys),
+        )
+
+        try:
+            destination_object_key = get_random_export_name(user_id)
+
+            await self.create_file_upload_links(
```

Comment:
what is the goal of this call? you do not use the return value.

Reply:
I've actually checked why it's like this.
 The return value is an object containing a link, but does not contain a `StorageFileID` object.